test/integration/scenarios/propagation/rpc/transactions/multisignature.js

```diff
@@ -141,10 +135,15 @@ module.exports = function multisignature(params) {
 			});
 
 			it('check all the nodes received the transactions', done => {
+				// Adding two extra blocks as a safety timeframe
 				const blocksToWait =
-					Math.ceil(MAXIMUM / constants.maxTransactionsPerBlock) + 1;
+					Math.ceil(MAXIMUM / constants.maxTransactionsPerBlock) + 2;
 				waitFor.blocks(blocksToWait, () => {
-					validateTransactionsOnAllNodes().then(done);
+					confirmTransactionsOnAllNodes(transactions, configurations)
```

Comment:
Is that function confirms transactions or wait for transactions to be confirmed?